rules_python_zip/simplepack.go

```diff
@@ -501,6 +502,14 @@ if need_unzip and isinstance(__loader__, zipimport.zipimporter):
     tempdir_create_pid = os.getpid()
     atexit.register(clean_tempdir_parent_only, tempdir)
     sys.path.insert(0, tempdir)
+    # Handle linux signal terminate by calling exit, so atexit code executes.
+    old_handler = None
+    def sig_exit(*args):
+        if sys.path[0].endswith('_pyzip'):
+            shutil.rmtree(sys.path[0])
```

Comment:
Duplicate `shutil` call -- why not just use the existing `clean_tempdir_parent_only` function... It handles the possibility of forking, which I don't believe your code does?

Reply:
Yup, that works too. Updated.